SchedulearnBackend/SchedulearnBackend/appsettings.json

```diff
@@ -9,5 +9,11 @@
     "ConnectionStrings": {
         "SchedulearnDatabase": "Data Source=(LocalDb)\\MSSQLLocalDB;Initial Catalog=SchedulearnBackend1;Integrated Security=SSPI;"
     },
-    "AllowedHosts": "*"
+    "AllowedHosts": "*",
+    "Jwt": {
+        "Key": "labasRytasVerySecretSchedulearnKey",
```

Comment:
This is for testing purposes only, but we will probably need some way to not push this key.